string object_name        # [object_name].plcf gives configuration
---
bool success
